src/npc.cpp

```diff
@@ -104,6 +104,36 @@ npc::npc()
     last_updated = calendar::turn;
 }
 
+standard_npc::standard_npc( const std::string &name, const std::vector<itype_id> &clothing,
+                            int sk_lvl, int s_str, int s_dex, int s_int, int s_per )
+{
+    this->name = name;
+    position = { 0, 0, 0 };
```

Comment:
Yes, it's better to leave it as it is here, but we'll have to make it right at some later point.
BTW looks like this magic number is responsible for the crash: `get_cache()` tries to get a level cache at index 510 which is out of bounds.